src/transformers/modeling_utils.py

```diff
@@ -1006,6 +1006,72 @@ def get_input_embeddings(self) -> nn.Module:
         else:
             raise NotImplementedError
 
+    def warn_if_pad_token_in_input_ids_no_attention_mask(self, input_ids, attention_mask):
```

Comment:
Let's not push descriptive names too far ;-)  I think `padding_attention_mask_warning` is more than enough!